runtime/hsa-runtime/core/driver/xdna/amd_xdna_driver.cpp

```diff
@@ -88,7 +92,12 @@ uint64_t XdnaDriver::GetDevHeapByteSize() {
   return dev_heap_size;
 }
 
-hsa_status_t XdnaDriver::Init() { return InitDeviceHeap(); }
+hsa_status_t XdnaDriver::Init() {
+  // Get timeout value from environment variable if present.
+  char *timeout_env_var_ptr = getenv(TIMEOUT_ENV_VAR);
```

Comment:
Do we need this timeout stuff upstream?

Reply:
Do you find it useful? I find it a nice way to give this control to the user, but if you don't think it will be used happy to remove it.

Reply:
If we do need it, there are some functions in utils for getting environment variables that abstract what the OS offers.

Reply:
I think we should leave this out of production code.

Reply:
Removed in c393105f70cdd24f4c3e81f10fc5111c28c197df.